src/Data/Vec/Membership/Propositional/Properties.agda

```diff
@@ -10,8 +10,8 @@ module Data.Vec.Membership.Propositional.Properties where
 
 open import Data.Fin.Base using (Fin; zero; suc)
 open import Data.Product as Prod using (_,_; ∃; _×_; -,_)
-open import Data.Vec hiding (here; there)
 open import Data.Vec.Relation.Unary.Any using (here; there)
+open import Data.Vec hiding (here; there)
```

Comment:
This breaks the 'alphabetical order' where no characters is less than any character. And also, there is no effective change. Please revert.

(Even better would be to remove the `hiding` and put in a proper `using`)